vng_api_common/notifications/api/views.py

```diff
@@ -46,6 +46,15 @@ def handle_notification(self, message):
 
 
 class NotificationView(NotificationBaseView):
+    action = 'create'
+    permission_classes = (AuthScopesRequired,)
+    required_scopes = {
+        'create': Scope(SCOPE_NOTIFICATIES_PUBLICEREN_LABEL),
+    }
+
+    def create(self, request, *args, **kwargs):
```

Comment:
is this needed?

Reply:
no. I can make it this way:
```python
class NotificationView(NotificationBaseView):
    permission_classes = (AuthScopesRequired,)
    action = 'post'
    required_scopes = {
        'post': Scope(SCOPE_NOTIFICATIES_PUBLICEREN_LABEL),
    }
...
```
But `post` is not an action in DRF,  so I added `create`

Reply:
Added as issue #28